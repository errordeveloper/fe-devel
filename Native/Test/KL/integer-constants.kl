/*
 *  Copyright 2010-2012 Fabric Engine Inc. All rights reserved.
 */

operator entry()
{
  report(Integer(2147483647));
  report(Integer(-2147483648));
}
