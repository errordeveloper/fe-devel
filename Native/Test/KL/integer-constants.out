integer-constants.kl:4:3: error: unary operator '-' not supported for expressions of type 'Size'
